spec/requests/api/v1/articles/show_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe 'GET /api/v1/articles', type: :request do
+  let!(:article) { create(:article) }
+
+  describe 'successfully' do
+    before do
+      get "/api/v1/articles/#{article.id}"
+    end
+
+    it 'should respond with 200 status' do
+      expect(response).to have_http_status 200
+    end
+
```

Comment:
test that the response contains the correct attributes that you have in the serializer

Reply:
3 new it blocks in show_spec